en/02_Developer_Guides/12_Search/01_Searchcontext.md

```diff
@@ -77,15 +76,12 @@ class MyDataObject extends DataObject
 }
 ```
 
-[notice]
+> [!WARNING]
+> In case you need multiple contexts, consider name-spacing your request parameters by using `FieldList->namespace()` on
+> the `$fields` constructor parameter.
+
 See the [SearchFilter](../model/searchfilters) documentation for more information about filters to use such as the
 `GreaterThanFilter`.
```

Comment:
Removed this from the callout block because it doesn't belong in one, and to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.